docs/cobbler-conf/settings-yaml.rst

```diff
@@ -206,21 +209,52 @@ default: Depending on your distro. See values below.
 grub2_mod_dir
 #############
 
-TODO
+The directory where Cobbler looks for GRUB modules that are required for "cobbler mkloaders".
+
+default: Depends on your distribution. See values below.
+
+* (open)SUSE: ``"/usr/share/grub2"``
+* Debian/Ubuntu: ``"/usr/lib/grub"``
+* CentOS/Fedora: ``"/usr/lib/grub"``
 
 syslinux_dir
 ############
 
-TODO
+The directory where Cobbler looks for syslinux modules that are required for "cobbler mkloaders".
+
+default: Depends on your distribution. See values below.
+
+* (open)SUSE: ``"/usr/share/syslinux"``
+* Debian/Ubuntu: ``"/usr/lib/syslinux/modules/bios/"``
+* CentOS/Fedora: ``"/usr/share/syslinux"``
 
 bootloaders_modules
 ###################
 
-TODO
+A list of all modules "cobbler mkloaders" includes when building grub loaders.
+Typically, a grub loader uses the modules for PXE or HTTP Boot.
+
+default: Omited for readablity, please refer to the `settings.yaml` file in our GitHub repository.
```

Comment:
Just a nitpick: It would be great if you could link to the settings file you are referring to here, if that's possible. This would make the process for the user a bit quicker.

Reply:
I think that this is not an easy fix as we would have to adjust the links for all branches manually at the moment. I will open a low-priority docs issue that will mention this after merging this PR. For now I would resolve this conversation to prioritize the performance SUSE-Bugzilla issue that this PR needs to be backported to.